ijanki/prometheus-bundle/0.9/config/packages/ijanki_prometheus.yaml

```diff
@@ -0,0 +1,22 @@
+ijanki_prometheus:
+    storage_adapter_service: TweedeGolf\PrometheusClient\Storage\ApcuAdapter
+    metrics_path: /metrics
+    collectors:
+        http_requests_total:
```

Comment:
You do not need all the default config here. Put a link to your documentation instead.

Reply:
The point of this opinionated bundle is to provide these default metrics at installation time.

Reply:
Sorry, I misread the code. You are actually creating collectors that you name and configure. Removing `http_requests_total`,  `http_request_duration_seconds` etc would mean we are removing services.

These are fine to leave here :+1:

Make sure to address my other comments and I'll be happy to merge.

Reply:
Ok i should have resolved the issues, please review.